app/src/components/OrderInformation.tsx

```diff
@@ -0,0 +1,153 @@
+import React, { Component } from 'react';
+import { createStyles, WithStyles } from '@material-ui/core/styles';
+import SwipeableDrawer from '@material-ui/core/SwipeableDrawer';
+import Divider from '@material-ui/core/Divider';
+import { withStyles } from '@material-ui/styles';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import bigInt from 'big-integer';
+import { Message } from '../models/OrderBook';
+import { Styles } from '../styles/OrderInformation';
+import { MESSAGE_TYPE_ENUM } from '../constants/Enums';
+import {
+    convertNanosecondsUTCToCurrentTimezone, getLocalTimeString,
+    nanosecondsToString,
+    splitNanosecondEpochTimestamp,
+} from '../utils/date-utils';
+
+const styles = createStyles(Styles);
+
+interface Props extends WithStyles<typeof styles>{
+    orderId : number,
+    quantity: number,
+    lastModified: string,
+    createdOn: string,
+    price: number,
+    messages: Array<Message>,
+}
+
+interface State {
+    right: boolean,
+}
+
+class OrderInformation extends Component<Props, State> {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            right: true,
+        };
+    }
+
+    renderToggleDrawer(side, open) {
+        return event => {
+            if (event && event.type === 'keydown' && (event.key === 'Tab' || event.key === 'Shift')) {
+                return;
+            }
+
+            this.setState({ right: true, [side]: open });
+        };
+    }
+
+    renderSlideList(side) {
+        const {
+            // eslint-disable-next-line no-unused-vars,@typescript-eslint/no-unused-vars
+            orderId, quantity, lastModified, createdOn, price, messages, classes,
+        } = this.props;
+        return (
+            <div
+                role={'presentation'}
+                onClick={this.renderToggleDrawer(side, false)}
+                onKeyDown={this.renderToggleDrawer(side, false)}
+            >
+                <Table className={classes.orderInfo}>
+                    <TableHead className={classes.orderIdHeader}>
+                        <h2>
+                            {'ID: '}
+                            {orderId}
+                        </h2>
+                    </TableHead>
+                    <TableRow>
+                        <TableCell>
+                            {'Quantity: '}
+                            {quantity}
+                        </TableCell>
+                    </TableRow>
```

Comment:
![image](https://user-images.githubusercontent.com/29107916/74610687-6f0a1d00-50c3-11ea-9955-6d273c85b036.png)

can all of these info be right-aligned on the same line as the descriptor ? what do you think?

Reply:
Good idea! i think that would look good

Reply:
This is what it looks like when i right align values. Lmk what you think and i will commit
<img width="375" alt="Screen Shot 2020-02-16 at 5 12 20 PM" src="https://user-images.githubusercontent.com/26652806/74613789-8525d680-50df-11ea-99dd-44a95d358f6c.png">

Reply:
Here is the differently aligned one as requested! 
<img width="370" alt="Screen Shot 2020-02-16 at 7 53 37 PM" src="https://user-images.githubusercontent.com/26652806/74616494-1011cb80-50f6-11ea-9227-12ab26401d62.png">